guidelines/old_styleguide.md

```diff
@@ -1,5 +1,7 @@
 # Mozilla L10n Style Guide English (us)
 
+> This document is still a draft and is not ready for review
```

Comment:
Need to remove this file or comment?

Reply:
This file needs to be removed. Will be removed with next commit